pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "schema-enforcer"
-version = "1.2.2"
+version = "1.3.0"
```

Comment:
Should this be a 2.0 version due to the pydantic version bump? Or are there no backwards-incompatible changes to the schema-enforcer API itself?

Reply:
that was my reasoning. I don't see any backwards-incompatible changes in the schema-enforcer. WDYT?

Reply:
Fine by me then.